cinema_service/settings.py

```diff
@@ -125,14 +122,17 @@
 
 USE_I18N = True
 
-USE_TZ = False
-
+USE_TZ = True
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = BASE_DIR / "media"
+
+MEDIA_URL = "/media/"
```

Comment:
The `MEDIA_ROOT` and `MEDIA_URL` settings are correctly configured to handle media files, which is essential for storing and serving uploaded images.